### PR TITLE
Gitcoin Bot Kudos - Skateboarder Inspired Bot

### PR DESCRIPTION
**Pull request by petitechoco**
Submitting a skateboarder bot towards https://github.com/gitcoinco/web/issues/4483

Screen Shot 2019-06-01 at 7 52 42 AM

### CODE REVIEW

**Comment by owocki** [comment]
woah how original.  its like a hoverboard!  does the bot have two sets of feet though?

**Comment by petitechoco** [comment]
No, those are arms and hands. Any suggestions?

**Comment by owocki** [comment]
i really like this piece of artwork, i just think it kind of looks like he's got four feet.   i wonder if we can make that those are hands not feet a little more clear.

**Comment by petitechoco** [comment]
Okay will make the changes and let you know shortly!

**Comment by petitechoco** [comment]
@owocki What do you think?
Screen Shot 2019-06-03 at 10 04 35 AM

File: Marketing/Art/Robots/skateboarder_bot/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+hi
```
Comment (owocki) [inline Marketing/Art/Robots/skateboarder_bot/readme.md]: i dont think this is needed

**Comment by owocki** [comment]
looks great . pls update the SVG on the PR

**Comment by petitechoco** [comment]
@owocki thx. updated 👍

**Pull request merged**